README.adoc

```diff
@@ -4,4 +4,21 @@ This plugin is under development.
 Please refer to https://www.jenkins.io/projects/gsoc/2020/projects/external-fingerprint-storage/[project page]
 for current status.
 
-The plugin aims to allow users to use Redis for storing fingerprints.
\ No newline at end of file
+The plugin aims to allow users to use Redis for storing fingerprints.
+
+== Quickstart
```

Comment:
Maybe call this "Quickstart for plugin developers"?

I think in the future people will install the plugin via plugin center in their Jenkins, no?

Reply:
I agree on "Quickstart for plugin developers". I will make the changes, thank you :)

Yeah users would be installing from there, I wanted to create this for anyone who would want to try it out before we have a release on the plugin center.

Reply:
NP, just wanted to save you and others some time in the future because this documentation will get out of stale and we might forget to update it :)

Reply:
Yes I agree, thank you so much! In that case I'll make the changes and leave it unmerged for the time being because I just want someplace I can refer anybody to for trying out https://github.com/jenkinsci/redis-fingerprint-storage-plugin/pull/2